api/openapi_server/controllers/auth_controller.py

```diff
@@ -217,7 +217,7 @@ def token():
     client_id = COGNITO_CLIENT_ID
     client_secret = COGNITO_CLIENT_SECRET
     callback_uri = request.args['callback_uri']
-    cognito_app_url = 'https://homeuudemo.auth.us-east-1.amazoncognito.com'
+    cognito_app_url = 'https://homeuniteus.auth.us-east-1.amazoncognito.com'
```

Comment:
sweet. couple questions as I'm still familiarizing myself with the code base,
- should we put this here or at the top with the other env variables?
- unit test? safety net that `requests.post(token_url, auth=auth, data=params)` on `233` is called with the updated url.

Reply:
Both are good ideas. I'll go ahead and it to the top. Maybe we want to make this an environment variable in the future? I don't think it's considered a secret, but I could see us potentially wanting to have dev/prod clients based on the environment. I'm not sure what our unit testing setup is like on the backend right now, but I like that idea as well.